File: length_of_last_word.py
class Solution:
    def lengthOfLastWord(self,s):
        lst=[]
        for word in s.split(' '):
            lst.append(word)
        for word in lst[::-1]:
            if len(word)>0:
                return len(word)
        return 0
     
if __name__=='__main__':
    sol=Solution()
    print sol.lengthOfLastWord('a ')

File: Algorithm/CPP/length_of_last_word.cpp
# include <vector>
# include <iostream>
using namespace std;
class Solution {
public:
	int lengthOfLastWord(char* s) {
		int len = 0;
		while (*s != '\0')
		{
			if (*s != ' ')
			{
				len++;
				s++;
			}
			else  {
				s++;
				if (*s && *s != ' ')
				{
					len = 0;
				}
			}	
		}
		return len;
	}
};
int  main()
{
	Solution sol;

	//nums = { -1 };
	char* s = " a";
	cout << sol.lengthOfLastWord(s);
	cin.get();
	return 0;
}
